Authorization/Authorization/Presentation/Registration/SignUpViewModel.swift

```diff
@@ -131,7 +131,7 @@ public class SignUpViewModel: ObservableObject {
                 fields: validateFields,
                 isSocial: externalToken != nil
             )
-            analytics.setUserID("\(user.id)")
+            analytics.identify(id: "\(user.id)", username: user.username, email: user.email)
```

Comment:
How about moving the `analytics.identify(.....)` call to `showMainOrWhatsNewScreen` so you don't have to worry about from where the main screen is being shown?  At the moment it's being called from multiple places.

Reply:
I don't really like the idea of passing User variable inside the router. However, we may come back to this question when we work with analytics (this is not part of this PR)

Reply:
ok